infrastructure/mockauth/resources/login.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>GC Digital Talent Mock Sign In</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
```

Comment:
Do we need all of these weights?

Reply:
Since it is a variabel font, it kind of comes with all the weights baked in. Though, I did make the call simpler in [reduce/fix fonts, add labels/required](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13076/commits/f679f77539894f6009755e3bdce80ac72cf7fb75)

Also, added labels/required to hopefully prevent the error :thinking:

Reply:
The `required` attribute does stop someone from easily getting to the error template with an empty form value, nice work. I was able to get there with removing the `required` attribute manually, but it seems unlikely anyone will make it that far unwillingly.

Reply:
Yeah, sadly we do not have access to the server side validation :cry: